src/commands/graph/graph-init.js

```diff
@@ -120,8 +108,32 @@ const graphInit = async (options, command) => {
     }
   }
 
-  if (envChanged) {
-    // Apply environment variable updates
+  if (!envChanged) {
+    log(`Graph-related environment variables already set for site ${siteInfo.name}`)
+    return true
+  }
+
+  // Apply environment variable updates
+
+  // eslint-disable-next-line unicorn/prefer-ternary
+  if (isUsingEnvelope) {
+    await api.createEnvVars({
+      accountId,
+      siteId,
+      body: [
+        !env.NETLIFY_GRAPH_WEBHOOK_SECRET && {
+          key: 'NETLIFY_GRAPH_WEBHOOK_SECRET',
+          scopes: ['functions'],
+          values: [{ context: 'all', value: newEnv.NETLIFY_GRAPH_WEBHOOK_SECRET }],
+        },
+        !env.NETLIFY_GRAPH_PERSIST_QUERY_TOKEN && {
+          key: 'NETLIFY_GRAPH_PERSIST_QUERY_TOKEN',
+          scopes: ['functions'],
```

Comment:
Are `scopes` typed? As in, will autocomplete ensure that I'm not putting in spurious scope values?

Also, this token needs to be available in build, please.

Reply:
Scopes are defined [here in the openapi spec](https://github.com/netlify/open-api/blob/master/swagger.yml#L257-L261) for this method -- should anything else be needed to populate autocomplete suggestions in the cli codebase?